Flea/Shared/ReservationForm.razor

```diff
@@ -94,11 +100,19 @@
         await UpdateReservationList();
     }
 
+    public void ResetSelectedCustomer()
+    {
+        _selectedCustomer = null!;
+        Reservation.ReservationOwner = null!;

```

Comment:
You don't need to set this since it doesn't affect rendering and a submissions should be blocked when `_selectedCustomer is null` anyways (of course only when it is a new reservation).

(There is no sound reasoning for using `null!` here)